qemu/tests/block_resize_unplug.py

```diff
@@ -103,4 +104,5 @@ def _check_img_size(size):
     if reboot:
         _change_vm_power()
         _check_vm_status()
+    win_driver_utils.memory_leak_check(vm, test, params)
```

Comment:
This should restrict ‘windows’ as this case also fit linux.
if is_windows:
    win_driver_utils.memory_leak_check(vm, test, params)

Reply:
Thanks, updated~